infra/modules/azure_function_app/variables.tf

```diff
@@ -155,3 +155,12 @@ variable "subnet_service_endpoints" {
   description = "(Optional) Enable service endpoints for the underlying subnet. This variable should be set only if function dependencies do not use private endpoints"
   default     = null
 }
+
+variable "alert_config" {
+  type = object({
+    action_group_id    = string
+    webhook_properties = map(string)
```

Comment:
what's the expected format for webhook_properties? can we add an example?

Reply:
I think it can be removed, nobody has ever used it: https://github.com/search?q=org%3Apagopa+webhook_properties+language%3AHCL&type=code&l=HCL

Reply:
removed